src/main/java/core/basesyntax/Pair.java

```diff
@@ -1,5 +1,67 @@
 package core.basesyntax;
 
-class Pair {
+class Pair<T1, T2> {
+    private T1 first;
+    private T2 second;
 
+    private Pair(T1 first, T2 second) {
+        this.first = first;
+        this.second = second;
+    }
+
+    public static <T1, T2> Pair of(T1 first, T2 second) {
+        return new Pair<>(first, second);
+    }
+
+    T1 getFirst() {
```

Comment:
```suggestion
    public T1 getFirst() {
```
and same with getSecond()